shell/platform/fuchsia/flutter/tests/integration/touch-input/touch-input-test.cc

```diff
@@ -492,28 +492,6 @@ TEST_P(FlutterEmbedTapTest, FlutterEmbedTap) {
   ASSERT_EQ(touch_injection_request_count(), 2);
 }
 
-TEST_P(FlutterEmbedTapTest, FlutterEmbedHittestDisabled) {
-  FML_LOG(INFO) << "Initializing scene";
-  AddComponentArgument("--no-hitTestable");
```

Comment:
We should delete the vestigial code that this is no longer exercising:  This means removing all code in https://github.com/flutter/engine/blob/main/shell/platform/fuchsia/flutter/tests/integration/touch-input/embedding-flutter-view/lib/embedding-flutter-view.dart which references "hitTestable".